Lib/gftools/builder/operations/fontc/__init__.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from typing import List
+from gftools.builder.operations import OperationBase
+
+FONTC_PATH = "FONTC_PATH_NOT_SET"
```

Comment:
_ prefix for a "private" thing.

Suggest default value of None.

Reply:
I chose this default value so that if somehow this ended up not getting set there would be a slightly more informative error message. Is that too clever?